app/keepers.go

```diff
@@ -605,7 +619,7 @@ func (app *NibiruApp) initAppModules(
 		),
 		auth.NewAppModule(appCodec, app.AccountKeeper, authsims.RandomGenesisAccounts, app.GetSubspace(authtypes.ModuleName)),
 		vesting.NewAppModule(app.AccountKeeper, app.BankKeeper),
-		bank.NewAppModule(appCodec, app.BankKeeper, app.AccountKeeper, app.GetSubspace(banktypes.ModuleName)),
+		bank.NewAppModule(appCodec, app.bankBaseKeeper, app.AccountKeeper, app.GetSubspace(banktypes.ModuleName)),
```

Comment:
This line's required to appease the module manager when it does a type assertion that the object of type `bankkeeper.Keeper` (an interface) is actually the `bankkeeper.BaseKeeper` struct. I didn't see a simple way to force x/bank to use the `NibiruBankKeeper` struct without editing the Bank module directly, so I opted for what's written here, which is much simpler.